profiler/src/ProfilerEngine/Datadog.Profiler.Native/OpSysTools.cpp

```diff
@@ -398,7 +398,7 @@ std::string OpSysTools::GetProcessName()
     return fs::path(pathName).filename().string();
 #elif MACOS
     const int32_t length = 260;
-    char* buffer = new char[length];
+    char* buffer[length] = {'\0'};
```

Comment:
It's good to avoid the memory leak but... char buffer[length] would be better  ;^)

Reply:
Good catch :)
thx